### DISCUSSION TITLE
TrackerControl

### DISCUSSION THREAD

**Discussion by ushilives** [Divested-Mobile/DivestOS-Build#118] [General] [answered]
Have you tried an app called TrackerControl? Can't be used with a private DNS. Just wondered what would be more secure, to use this app or use a private DNS.

**Comment by SkewedZeppelin** [comment, marked as answer]
The list included with DivestOS is far more effective and doesn't interfere with
Private DNS or take your VPN slot.

**Reply by ushilives** [reply]
OK thanks for speedy reply! I will use this ROM for Poco F1, shame camera doesn't work on it yet but I will change kernel to fix the camera
